src/Worker.h

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright (c) 2021 Chiu Yen-Chen, Swen Sun-Yen, Wen Yong-Wei, Yuan Wei-Chen.
+All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file. See the AUTHORS file for names of contributors.
+ */
+
+#ifndef SHORTLINK_SRC_WORKER_H_
+#define SHORTLINK_SRC_WORKER_H_
+#include <string>
+
+namespace shortlink
+{
+    enum worker_status
+    {
+        wBusy,
```

Comment:
Please name according to:
https://google.github.io/styleguide/cppguide.html#Enumerator_Names